bedrock/mozorg/templates/mozorg/contribute.html

```diff
@@ -8,7 +8,12 @@
 
 {% extends "base-protocol-mozilla.html" %}
 
+{% set show_mozilla_connect_banner = switch('mozilla-connect-banner') %}
```

Comment:
[I checked if Konstantina wanted to restrict the banner ](https://github.com/mozilla/bedrock/issues/11233#issuecomment-1059108108)to show only when translations are available and this was her reply: 

> It's ok if it's not translated, the platform isn't translated anyways so that's alright

Do we always restrict content until translations are available? I can go back and let the stakeholders know if we need to restrict content as part of our standard l10n practice.

Reply:
I also had a go at adding a Learn More button instead of linking the title but it feels a bit off (the button is using mzp-t-md for size). I'm inclined to leave the title link as-is.

<img width="1236" alt="Screen Shot 2022-03-14 at 11 26 29 AM" src="https://user-images.githubusercontent.com/19650432/158165564-82a71de5-0987-4b5e-a0d9-a1b36e633a59.png">

Reply:
> Do we always restrict content until translations are available? I can go back and let the stakeholders know if we need to restrict content as part of our standard l10n practice.

Typically we try to avoid mixed-language content, but in the case of this banner maybe it's fine if we show it in English even when the rest of the page is translated. It's only a small bit of text and is temporary anyway.